### ISSUE TITLE
修改符号表后，如何变更模型

### ISSUE THREAD

**Issue by shuxiang** [RVC-Boss/GPT-SoVITS#2615]
修改符号后 [参考地址](https://github.com/RVC-Boss/GPT-SoVITS/wiki/%E8%AE%AD%E7%BB%83%E6%96%B0%E8%AF%AD%E8%A8%80(how-to-train-the-models-with-other-languages))

之后要修改模型，但我有个疑问，比如V3，有预训练SoVITS-G模型路径、预训练SoVITS-D模型路径、预训练GPT模型路径，是否三个都要修改？
下面代码中 new_weight = torch.cat([first_part,second_part, torch.randn(2, 512)],dim=0)  new_weight = torch.cat([first_part,torch.zeros(2,192),second_part],dim=0) 的 512和192 是否也要修改？这两个数字含义是什么？

参考代码：
`
import torch
#### 迁移gpt模型的embedding层,插入两个新的token
dict = torch.load(r"D:\pyprojs\GPT-SoVITSs\fork\GPT-SoVITS\GPT_SoVITS\pretrained_models\s1bert25hz-2kh-longer-epoch=68e-step=50232-2.ckpt",map_location=torch.device('cpu'))
dict["weight"]["model.ar_text_embedding.word_embeddings.weight"].shape
# 0~94保持不动，95~向后移动两位，新的95 96随机初始化
# 一共512列，embedding_dim=5
first_part = dict["weight"]["model.ar_text_embedding.word_embeddings.weight"][:95,:]
second_part = dict["weight"]["model.ar_text_embedding.word_embeddings.weight"][95:-2,:]
new_weight = torch.cat([first_part,second_part, torch.randn(2, 512)],dim=0)
dict["weight"]["model.ar_text_embedding.word_embeddings.weight"] = new_weight
torch.save(dict,r"D:\pyprojs\GPT-SoVITSs\fork\GPT-SoVITS\GPT_SoVITS\pretrained_models\s1bert25hz-hs_model.ckpt")
import torch
dict2 = torch.load(r"D:\pyprojs\GPT-SoVITSs\fork\GPT-SoVITS\GPT_SoVITS\pretrained_models\s2G488k-legacy.pth",map_location=torch.device('cpu'))
dict2["weight"]["enc_p.text_embedding.weight"].shape
# 0~94保持不动，95~向后移动两位，新的95 96随机初始化
first_part = dict2["weight"]["enc_p.text_embedding.weight"][:95,:]
second_part = dict2["weight"]["enc_p.text_embedding.weight"][95:,:]
new_weight = torch.cat([first_part,torch.zeros(2,192),second_part],dim=0)
dict2["weight"]["enc_p.text_embedding.weight"] = new_weight
torch.save(dict2,r"D:\pyprojs\GPT-SoVITSs\fork\GPT-SoVITS\GPT_SoVITS\pretrained_models\s2G488k.pth")
`
